httprequest_lego_provider/views.py

```diff
@@ -2,15 +2,20 @@
 # See LICENSE file for licensing details.
 """Views."""
 
+# pylint:disable=too-many-ancestors
```

Comment:
Perhaps add a small comment (I assume its because we have no control over class inheritance)